### DISCUSSION TITLE
“Document Storage” Option in Settings App?

### DISCUSSION THREAD

**Discussion by WinkelCode** [holzschu/a-shell#574] [Q&A] [answered]
Hi, I noticed there is a "Document Storage" option in the Settings app section for a-Shell. I know that a-Shell automatically creates a self-titled folder in iCloud, but I never figured out what exactly goes there.

Anyone know more about this?

**Comment by holzschu** [comment, marked as answer]
It’s an option added by Apple to every app. I haven’t found what it does, or even how to access the value of that settings.

**Reply by WinkelCode** [reply]
> It’s an option added by Apple to every app. I haven’t found what it does, or even how to access the value of that settings.

Of the apps on my device I can only find this option for iDOS, Procreate, and Textastic. It seems to list all the enabled Files.app "locations".
Is the automatically created a-Shell folder in iCloud related to this? The other apps mentioned above didn't generate such a folder.

**Edit:** This also doesn't appear to be related to apps creating folders, Möbius Sync, VLC, and the Affinity Suite have neither the option, nor the automatically created iCloud Folder. The only other such folder I have is from "Shortcuts", but it's empty since Shortcuts are saved using the "hidden" iCloud syncing mechanism that doesn't show up in Files.

**Comment by Shreebz** [comment]
How do you make a-Shell use the iCloud destination? Mine is using the "On My iPhone" location, and I would like the a-Shell app folder to use iCloud.

**Comment by holzschu** [comment]
The short answer is that I don't know. You can always put "cd ~cloud" in your `.profile` file, and a-Shell will start in the iCloud folder, but all the important configuration files will still be at "On My iPhone".
